includes/classes/blocks/class-bindings.php

```
@@ -559,23 +559,28 @@ public function build_unit_field( $build = '', $field = '', $count = 1 ) {
-				}
-
-				$value = $currency . $value;
-
-				// Get the currency settings
-				if ( is_object( $tour_operator ) && ( isset( $tour_operator->options['country_code_disabled'] ) && 0 === intval( $tour_operator->options['country_code_disabled'] ) || ! isset( $tour_operator->options['country_code_disabled'] ) ) ) {
-					$value = $letter_code . $value;
-				}
-
-				$pattern = '/(<p\s+[^>]*\bclass="[^"]*\bunit-price\b[^"]*"[^>]*>).*?(<\/p>)/is';
-				break;
+				$raw_price = $rooms->item_price( '', '', false );
```

PHPMD (2.15.0)

The variable $raw_price is not named in camelCase. (undefined)

(CamelCaseVariableName)